packages/ui-stencil/src/components/internal/orama-sliding-panel/orama-sliding-panel.scss

```diff
@@ -1,6 +1,39 @@
 $height-panel: 95vh;
 $height-panel-header: 5vh;
 
+.slide-backdrop {
+  &.visible {
+    display: none;
+    position: fixed;
+    display: block;
+    width: 100vw;
+    height: 100vh;
+    top: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.8);
```

Comment:
this backdrop color has to be alway equal to the modal backdrop, so we can be sure that every color customization can be done safely. We can replace with:
`background-color: var(--backdrop-background-color-primary, palette('backdrop-background', 'primary'));`